frontend/.env.production

```diff
@@ -1,7 +1,6 @@
 GATSBY_APP_NAME=Blockipedia
 GATSBY_HOSTNAME=https://nlhkh.github.io
 GATSBY_PATH_PREFIX=/blockipedia-near
-GATSBY_ROOT=https://nlhkh.github.io/blockipedia-near
```

Comment:
Would this break Github Pages deployment?

Reply:
should not break, cuz in `gatsby.config` we're populating `siteUrl` by concatenating `GATSBY_HOSTNAME` and `GATSBY_PATH` now, and so `GATSBY_ROOT` can be considered deprecated :)

Reply:
Anyhow, ill keep watch of the GitHub Pages deployment after merging. If it's ever broken, I'll patch it up :)